app/models/micropost.rb

```diff
@@ -1,8 +1,11 @@
 class Micropost < ApplicationRecord
   belongs_to :user
   mount_uploader :picture, PictureUploader
-  scope :by_newest, -> { order(created_at: :desc) }
+  scope :by_newest, ->{order created_at: :desc}
   default_scope {by_newest}
```

Comment:
You shouldn't use `default_scope`, because it's anti pattern
http://weblog.jamisbuck.org/2015/9/19/default-scopes-anti-pattern.html
Let using `scope` instead

Reply:
Yes